src/applications/disability-benefits/all-claims/pages/recentJobApplications.js

```diff
@@ -0,0 +1,109 @@
+import { merge, omit } from 'lodash';
+import fullSchema from '../config/schema';
+
+const { date } = fullSchema.definitions;
+
+import {
+  recentJobApplicationsDescription,
+  substantiallyGainfulEmployment,
+} from '../content/recentJobApplications';
+
+import {
+  uiSchema as addressUI,
+  schema as addressSchema,
+} from '../../../../platform/forms/definitions/address';
+
+const address = addressSchema(fullSchema);
+
+import { validateZIP } from '../validations';
```

Comment:
Should we move all of the imports above the `const` ? That way they are all relatively in the same location?